src/py/aspen/flask_app/flask_app.py

```diff
@@ -0,0 +1,47 @@
+import os
```

Comment:
can we name this src/py/aspen/app/app ?

Reply:
probably.  we'd need to change our entry point as well.

Reply:
i just think the flask part is unnecessary, but if it's too annoying i don't mind waiting until later

Reply:
no problem.